Towny/src/main/java/com/palmergames/bukkit/towny/listeners/TownyEntityListener.java

```diff
@@ -997,4 +1005,41 @@ private boolean hasDetrimentalEffects(Collection<PotionEffect> effects) {
                 return legacyName != null && detrimentalPotions.contains(legacyName);
             });
 	}
+
+	private static final @NotNull NamespacedKey GOLEM_CHEST_X = Objects.requireNonNull(NamespacedKey.fromString("towny:golemchestx"));
+	private static final @NotNull NamespacedKey GOLEM_CHEST_Z = Objects.requireNonNull(NamespacedKey.fromString("towny:golemchestz"));
+
+	@EventHandler(priority = EventPriority.NORMAL, ignoreCancelled = true)
+	public void onGolemMoveItem(ItemTransportingEntityValidateTargetEvent event) {
+		if (plugin.isError() || !TownyAPI.getInstance().isTownyWorld(event.getEntity().getWorld()) || !(event.getEntity() instanceof CopperGolem golem))
+			return;
+
+		// The golem is attempting to remove an item from a copper chest.
+		if (event.getBlock().getType().name().endsWith("COPPER_CHEST")) {

```

Comment:
A new copper chests item list would be nicer and ever so slightly faster than string checks